tests/test.py

```diff
@@ -425,24 +425,24 @@ def test_save_ml_output(self):
 class TestArrayMath(unittest.TestCase):
     
     
-    def test_scale_arrays(self):
-        sample_read= Poly5Reader(sample_emg)
-        sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 500)
-        new_emg = scale_arrays(sample_emg_filtered , 3,0)
-        self.assertEqual(
-            (new_emg.shape),
-            (sample_emg_filtered.shape),
-        )
+    # def test_scale_arrays(self):
+    #     sample_read= Poly5Reader(sample_emg)
+    #     sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 500)
+    #     new_emg = scale_arrays(sample_emg_filtered , 3,0)
+    #     self.assertEqual(
+    #         (new_emg.shape),
+    #         (sample_emg_filtered.shape),
+    #     )
 
-    def test_zero_one_for_jumps_base(self):
-        sample_read= Poly5Reader(sample_emg)
-        sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 500)
-        new_emg = zero_one_for_jumps_base(sample_emg_filtered[0] , sample_emg_filtered[0].mean())
-        new_emg = np.array(np.vstack((new_emg, new_emg)))
-        self.assertEqual(
-            (new_emg.shape[1]),
-            (sample_emg_filtered.shape[1]),
-        )
+    # def test_zero_one_for_jumps_base(self):
+    #     sample_read= Poly5Reader(sample_emg)
+    #     sample_emg_filtered = emg_bandpass_butter(sample_read, 1, 500)
+    #     new_emg = zero_one_for_jumps_base(sample_emg_filtered[0] , sample_emg_filtered[0].mean())
+    #     new_emg = np.array(np.vstack((new_emg, new_emg)))
+    #     self.assertEqual(
+    #         (new_emg.shape[1]),
+    #         (sample_emg_filtered.shape[1]),
+    #     )
 
     def test_find_peaks_in_ecg_signal(self):
```

Comment:
I think it is best to move this test along with the rest of the test class. I suggest to move its part that tests find_peaks_in_ecg_signal to the the preprocessing tests. THen 425 can also be commented out.